services/gem/gem-version.tester.js

```diff
@@ -11,3 +14,19 @@ t.create('version (valid)').get('/formatador.json').expectBadge({
 t.create('version (not found)')
   .get('/not-a-package.json')
   .expectBadge({ label: 'gem', message: 'not found' })
+
+// this is the same as isVPlusDottedVersionNClausesWithOptionalSuffix from test-validators.js
+// except that it also accepts regexes like 5.0.0.rc5 - the . before the rc5 is not accepted in the original
+const isVPlusDottedVersionNClausesWithOptionalSuffix = withRegex(
```

Comment:
I'm wondering whether it would have been better to modify the original regex directly. Admittedly I've not seen many projects use a dot as the separator for the pre-release tag outside the Ruby ecosystem, so what you've done may make more sense. Let's leave it as is for now, but another maintainer will possibly have a stronger view on this one. :)

Reply:
generally agreed but also content with leaving this here for now. we probably do have some minor proliferation of duplicative regexes but I don't have any concerns with having this one here